### ISSUE TITLE
bug(windows): searching for a language from the Add Language to Keyboard

### ISSUE THREAD

**Issue by MakaraSok** [keymanapp/keyman#4203]
**Describe the bug**

It looks like not any language name can be searched for in the language field. I would like to search for "Mnong". Typing "Mnong" in the field would not yield the expected language, unless typing in "cmo".

When typing in "cmo":
![image](https://user-images.githubusercontent.com/28331388/102847604-4bcb9f00-4445-11eb-9836-04a3996fc2f9.png)

When typing in "Mnong":
![image](https://user-images.githubusercontent.com/28331388/102847862-f5ab2b80-4445-11eb-9f3d-ac60aa9abd47.png)

**To Reproduce**

1. Go to 'Configuration'
2. Click on a keyboard, say 'Khmer Angkor' to expand
3. Typing in the language field "cmo" and then "Mnong"
4. See the different results

**Expected behavior**

"Mnong" in the language field should yield the expected language as that is probably more intuitive to the users. Typing its alternate name "Bunong" should also yield the language.

---

**Keyman for Windows:**
 - OS: Windows 10 Pro 1909 x64
 - Keyman version: 14.0.214-beta
 - Target Application: Configuration

**Comment by mcdurdin** [comment]
This dialog currently supports searching only a small subset of language names -- if you are searching for a language that is not in the unfiltered list, you have to enter the language code.

This would be an additional feature which is a fair bit of work; not sure how valuable it is.

**Comment by ermshiperete** [comment]
> This dialog currently supports searching only a small subset of language names -- if you are searching for a language that is not in the unfiltered list, you have to enter the language code.

"Khmer (SIL)" lists cmo-Khmr as supported language, but searching for "Mnong" still doesn't show it. Searching for "Central Mnong" though works.

It seems the dialog lists the languages that the keyboard supports, plus the languages the OS knows about.

**Comment by mcdurdin** [comment]
> It seems the dialog lists the languages that the keyboard supports, plus the languages the OS knows about.

Correct. And the search is fairly simplistic because it won't do substring searches at present, I think.
